dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/FeaturesContextResolver.kt

```diff
@@ -7,16 +7,27 @@
 package com.datadog.android.rum.internal
 
 import com.datadog.android.v2.api.Feature
-import com.datadog.android.v2.api.context.DatadogContext
+import com.datadog.android.v2.api.SdkCore
 
 internal class FeaturesContextResolver {
 
-    fun resolveHasReplay(context: DatadogContext, viewId: String): Boolean {
+    @Suppress("UNCHECKED_CAST")
+    fun resolveViewHasReplay(sdkCore: SdkCore, viewId: String): Boolean {
```

Comment:
why do we prefer now getting it from `SdkCore` and not from `DatadogContext`? To me it seems in all the call sites we can (and we should) read it from `DatadogContext`.